cmd/link-client/main.go

```diff
@@ -150,6 +151,52 @@ func main() {
 				fmt.Println(aurora.Green(fmt.Sprintf("IP %s (%s) successfully added", newIP.IP.IP, newIP.ID)))
 				return nil
 			},
+		}, {
+			Name:      "update-healthchecks",
+			ArgsUsage: "ID [CHECK_TYPE CHECK_ENDPOINT]...",
+			Action: func(c *cli.Context) error {
+				if len(c.Args())%2 == 0 {
```

Comment:
[nitpicking] Why not directly use `c.NArg` as written in the comment and the for loop ?